### DISCUSSION TITLE
Is possible to ignore code block in markdown?

### DISCUSSION THREAD

**Discussion by F3n67u** [znck/grammarly#282] [Q&A]
Most of the Grammarly error exists in my markdown document's code block which causes a lot of noise, could we ignore the error on the code block(i.e. the block surrounded by ``` and `)?

<img width="788" alt="image" src="https://user-images.githubusercontent.com/12343178/171986592-3c76645c-eea4-4388-a960-89caf60b907d.png">

**Comment by ahmad-ali14** [comment]
yeah, it might be a good idea to ignore everything in between backticks, but then the whole sentence will change and the Grammarly server will definitely find an error in such a sentence.

**Comment by santaimpersonator** [comment]
In the old version, you could ignore the code blocks, but that setting seems to have disappeared with the newest release.

![feature_enable](https://user-images.githubusercontent.com/36016723/185512585-3253b4d4-f3e2-4d8f-bc60-bc17a107129c.PNG)
